lms/djangoapps/bulk_email/tasks.py

```diff
@@ -95,29 +95,35 @@
 )
 
 
-def _get_course_email_context(course):
+def _get_course_email_context(course, site_context):
     """
     Returns context arguments to apply to all emails, independent of recipient.
     """
     course_id = text_type(course.id)
     course_title = course.display_name
     course_end_date = get_default_time_display(course.end)
     course_root = reverse('course_root', kwargs={'course_id': course_id})
+    base_url = site_context.get('LMS_ROOT_URL', settings.LMS_ROOT_URL) if site_context \
```

Comment:
Assuming we applied the previous suggestion, we could do:
```suggestion
    base_url = site_context.get('LMS_ROOT_URL', settings.LMS_ROOT_URL) 
```